### PR TITLE
Animations: 80% of animations 

### PR DESCRIPTION
**Pull request by jzhzhang66**
This PR contains: 
- Submit / resource button on hover / on tap animations 
- Popup fade in animation / popup close icon hover state
- Home page mobile and desktop logo animations 
- Story preview (home and archive page) hover and click states
- Header z-index bug fix 

Animations not done yet / struggling with: 
- How to make home page animations after first hover smoother 
- How to make current featured story info fade in and out 
- Intersection observer / getting text to fade in with scroll

### CODE REVIEW

File: components/home-page/HomePage.js [hunk 1]
```diff
@@ -36,9 +43,14 @@ export default function HomePage({
...
   };
 
+  const isMobile = useMatchMedia(`(max-width: ${breakpointsObj.tabletSm}px)`);
   const getSignature = (authorInfoID) => authorSignatures[String(authorInfoID)];
-
   const [activeIndex, setActiveIndex] = useState(0);
+  const [fadedLong, setFadedLong] = useState(false);
+  const [fadedShort, setFadedShort] = useState(false);
+
+  setTimeout(() => setFadedLong(true), 6000);
```
Comment (isabellberzsenyi) [inline components/home-page/HomePage.js]: Can we make the desktop animation slightly shorter

File: components/home-page/current-featured-story/CurrentFeaturedStory.styles.js [hunk 2]
```diff
@@ -7,15 +7,21 @@ import {
...
   max-width: 1050px;
   padding: min(7%, 80px);
   padding-bottom: 0px;
-
+  @media ${max.tabletLg} {
+    padding-top: 5em;
+  }
+  @media ${max.tablet} {
+    padding-top: 5em;
+  }
```
Comment (isabellberzsenyi) [inline components/home-page/current-featured-story/CurrentFeaturedStory.styles.js]: nit: line 19-21 does the same, so you can remove these lines

File: components/story-archive-page/archive-story-preview/ArchiveStoryPreview.js [hunk 3]
```diff
@@ -29,22 +29,28 @@ export default function ArchiveStoryPreview({ story, signature }) {
   const isTabletOrMobile = useMatchMedia(
     `(max-width: ${breakpointsObj.tabletLg}px)`,
   );
-
+  const [hover, setHover] = useState(false);
+  // initial={{opacity: 0}} animate={{opacity: 1}} transition={{duration: 1}}
```
Comment (isabellberzsenyi) [inline components/story-archive-page/archive-story-preview/ArchiveStoryPreview.js]: nit: do we need this?

**Review by isabellberzsenyi — APPROVED** [review]
couple nitpicks looks great

File: components/four-oh-four-page/FourOhFourPage.style.js [hunk 4]
```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components';
 import { H2, P } from '../../style/typography';
 import { max } from '../../lib/responsive';
+import { animated } from 'react-spring';
 
-const ImgBackground = styled.img`
+const ImgBackground = styled(animated.img)`
```
Comment (josephmannis) [inline components/four-oh-four-page/FourOhFourPage.style.js:6]: Are you intending to mix the two libraries?

**Review by josephmannis — APPROVED** [review]
IDK this seems stinkin good to me

File: components/four-oh-four-page/FourOhFourPage.style.js (hunk 4 above)
Comment (jzhzhang66) [reply components/four-oh-four-page/FourOhFourPage.style.js:6]: Yep!

**Pull request merged**